docs/user/developers/compiling.rst

```diff
@@ -74,157 +68,134 @@ repository::
+Create signatures for binaries
+==============================
 
-.. warning::
-  These instructions assume that a PGP key for <signer> exists on the build
-  system. If the expected key is not found, the script will fail at the signing
-  step with a message including::
+Mac and Windows binaries are signed by Dash Core Group using the relevant Apple
+and Microsoft processes. In this step, that information will be validated and
+signed by your machine. 
 
-    gpg: skipped "<signer>": No secret key
-    gpg: signing failed: No secret key
+Prepare the `detached sigs repository <https://github.com/dashpay/dash-detached-sigs>`__::
 
-When the build completes, it will put the binaries in a ``dashcore-binaries``
-folder. The ``.assert`` files and their signatures will be placed in
-``gitian.sigs/<version>/<signer>/...``.
+  cd ~/dash-detached-sigs/
+  git checkout master
+  git pull
+  # Checkout the branch for the version being built
+  # Example: git checkout v20.0.3
+  git checkout v<version>
 
-Create signatures for signed binaries
--------------------------------------
+Prepare the `guix.sigs repository <https://github.com/dashpay/guix.sigs>`__ by
+pulling the latest changes::
 
-Mac and Windows binaries are signed by Dash Core Group using the relevant
-Apple/Microsoft processes. In this step, that information will be validated and
-signed by your machine. The associated ``.assert`` files and their signatures
-will be placed in ``gitian.sigs/<version>/<signer>/...`` along with the
-signatures for unsigned binaries created in the previous step.
+  cd ~/guix.sigs/
+  git checkout master
+  git pull
 
-::
+Create a new branch for the version that was built::
 
   # <signer> = The name associated with your PGP key
   # <version> = Dash Core tag to build (exclude the leading "v")
-  # Example: ./dash/contrib/gitian-build.py -s -n -j $(nproc) -m 16000 -o mw alice 0.17.0.3
-  ./dash/contrib/gitian-build.py -s -n -j $(nproc) -m <MB of RAM to use> -o mw <signer> <version> 
-
-Verify signatures
-=================
+  # Example: git checkout -b 20.0.3-alice
+  git checkout -b <version>-<signer>
 
-The `gitian.sigs repository <https://github.com/dashpay/gitian.sigs/>`_ contains
-deterministic build results signed by multiple Core developers for each release.
-Run the following command to verify that your build matches the official
-release::
+Unsigned binaries
+-----------------
 
-  # Example: ./dash/contrib/gitian-build.py -v alice 0.17.0.3
-  ./dash/contrib/gitian-build.py -v <signer> <version>
+To create signatures for the unsigned binaries, run ``guix-attest``::
 
-You should get a result similar to the following for Linux, Windows, MacOS,
-Signed Windows, and Signed MacOS. Assuming the previous steps completed
-successfully, you will also see your own signatures with an ``OK`` status also.
+  # <signer> = The name associated with your PGP key
+  # Example: env GUIX_SIGS_REPO=~/guix.sigs SIGNER=alice ./contrib/guix/guix-attest
+  cd ~/dash
```

Comment:
do we need here `cd ~/dash`? if so, need to add also below where's "Signed binaries"

Reply:
It's need here because the previous step would leave you in the `guix-sigs` directory. If you're going through the steps sequentially, you will already be in the `dash` directory when you get to "Signed binaries" next.